pkg/cmd/server/kubernetes/master.go

```diff
@@ -223,7 +223,7 @@ func probeRecyclableVolumePlugins(config componentconfig.VolumeConfiguration, na
 	nfsConfig := volume.VolumeConfig{
 		RecyclerMinimumTimeout:   int(config.PersistentVolumeRecyclerConfiguration.MinimumTimeoutNFS),
 		RecyclerTimeoutIncrement: int(config.PersistentVolumeRecyclerConfiguration.IncrementTimeoutNFS),
-		RecyclerPodTemplate:      volume.NewPersistentVolumeRecyclerPodTemplate(),
+		RecyclerPodTemplate:      defaultScrubPod,
```

Comment:
@liggitt This change is correct.  The default was already set on line 194 above and the overridden, so  this line should be the `defaultScrubPod` variable.